example-2.0/server/model-config.json

```diff
@@ -26,6 +26,6 @@
   },
   "container": {
     "dataSource": "storage",
-    "public": true
+    "public": false
```

Comment:
I don't think if we should hide `contianer` model in explorer. @bajtos @superkhau thought?